source/consts/application.ts

```diff
@@ -0,0 +1,6 @@
+declare const __VERSION__: string;
```

Comment:
It was good for backtrace-js because we use webpack to build the library. Here we don't so I believe we don't need to have __VERSION__